lua/nvim-tree/explorer/init.lua

```diff
@@ -664,15 +667,15 @@ end
 ---@param node Node
 function Explorer:dir_up(node)
   if not node or node.name == ".." then
-    change_dir.fn("..")
+    self:change_dir("..")
   else
     local cwd = core.get_cwd()
     if cwd == nil then
       return
     end
```

Comment:
We can avoid this core call now - we always know we have the `Explorer` instance so can use `self.absolute_path`